4-WebApp-your-API/4-1-MyOrg/TodoListService/Startup.cs

```diff
@@ -26,7 +27,7 @@ public void ConfigureServices(IServiceCollection services)
             // By default, the claims mapping will map claim names in the old format to accommodate older SAML applications.
             // 'http://schemas.microsoft.com/ws/2008/06/identity/claims/role' instead of 'roles'
             // This flag ensures that the ClaimsIdentity claims collection will be built from the claims in the token
-            // JwtSecurityTokenHandler.DefaultMapInboundClaims = false;
+            JwtSecurityTokenHandler.DefaultMapInboundClaims = false;
 
             // Adds Microsoft Identity platform (AAD v2.0) support to protect this Api
             services.AddMicrosoftIdentityWebApiAuthentication(Configuration);
```

Comment:
>AddMicrosoftIdentityWebApiAuthentication [](http://example.com/codeflow?start=21&length=40)

add one line of extended validation that verifies tenantId (its redundant but shows usage.)

Reply:
added